components/stepped-form/index.tsx

```diff
@@ -168,14 +168,24 @@ interface FormField {
   values?: any[]; // optional values to select
 }
 
+interface FormData {
+  id: number
+  active: boolean
+  createdBy: number
+  updatedBy: number
+  createdOn: Date
+  updatedOn: Date
+  sections: Array<object>
+}
+
 const SteppedForm = () => {
-  const [formData, setFormData] = useState<object>({});
+  const [formData, setFormData] = useState<FormData>();
   const [loading, setLoading] = useState<boolean>(true);
 
-  const initForm = () => {
+  const initForm = async () => {
     setLoading(true);
     const getFormData = getFormTemplate("1", "1");
-    setFormData(getFormData);
+    setFormData(await getFormData);
```

Comment:
this is not where await would go, I think you would put it in front of getFormTemplate so "= await getFormTemplate"

Reply:
either works - depends if you want getFormData to explicitly be the response object or a Promise - in this case was just fixing a linter warning